recorder/src/main/cpp/controller.cc

```diff
@@ -416,15 +429,15 @@ void populate_recording_header(recording::RecordingHeader& rh, const recording::
     *wa = w;
 }
 
-void Controller::issueWork(const std::string& host, const std::uint32_t port, std::uint32_t controller_id, std::uint32_t controller_version) {
+void Controller::issue_work(const std::string& host, const std::uint32_t port, std::uint32_t controller_id, std::uint32_t controller_version) {
     auto at = Time::now() + Time::sec(current_work.delay());
     scheduler.schedule(at, [&, port, controller_id, controller_version]() {
             with_current_work([&](Controller::W& w, Controller::WSt& wst, Controller::WRes& wres, Time::Pt& start_tm, Time::Pt& end_tm) {
                     auto work_id = w.work_id();
```

Comment:
work-id is a dynamic field, metrics on it will not be that useful, actually.

Metric, tag and tag-values are supposed to be static values for both performance-reasons on TSDB side and for it to be useful on the dashboard (or for alerting).

Dynamic values will just fan-out real bad and get interpolated in very bad ways + will have too high cardinality (for eg. imagine work_id cardinality) for it to be loadable on the browser.